packages/react-core/src/components/HelperText/__tests__/HelperText.test.tsx

```diff
@@ -1,66 +1,66 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import { HelperText } from '../HelperText';
-import { HelperTextItem } from '../HelperTextItem';
-import { CheckIcon } from '@patternfly/react-icons';
 
-describe('HelperText', () => {
-  test('simple helper text renders successfully', () => {
-    const { asFragment } = render(
-      <HelperText>
-        <HelperTextItem>help test text</HelperTextItem>
-      </HelperText>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+import styles from '@patternfly/react-styles/css/components/HelperText/helper-text';
 
-  Object.values(['default', 'indeterminate', 'warning', 'success', 'invalid']).forEach(variant => {
-    test(`${variant} helper text variant applies successfully`, () => {
-      const { asFragment } = render(
-        <HelperTextItem variant={variant as 'default' | 'indeterminate' | 'warning' | 'success' | 'error'}>
-          {variant} help test text
-        </HelperTextItem>
-      );
-      expect(asFragment()).toMatchSnapshot();
-    });
-  });
+test('Renders to match snapshot', () => {
+  const { asFragment } = render(<HelperText />);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test('Renders without children', () => {
+  render(
+    <div data-testid="container">
+      <HelperText />
+    </div>
+  );
+
+  expect(screen.getByTestId('container').firstChild).toBeVisible();
+});
 
-  test('variant comonent helper text renders properly', () => {
-    const { asFragment } = render(
-      <HelperText component="ul">
-        <HelperTextItem component="li">help test text 1</HelperTextItem>
-        <HelperTextItem component="li">help test text 2</HelperTextItem>
-      </HelperText>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('Renders default classes', () => {
+  render(<HelperText>text</HelperText>);
+  expect(screen.getByText('text')).toHaveClass(styles.helperText);
+});
+
+test('Renders id when id is passed', () => {
+  render(<HelperText id="helper-id">text </HelperText>);
+  expect(screen.getByText('text')).toHaveAttribute('id');
+});
 
-  test('icon helper text renders properly', () => {
-    const { asFragment } = render(
-      <HelperText>
-        <HelperTextItem icon={<CheckIcon />}>help test text</HelperTextItem>
-      </HelperText>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('Renders aria-live when isLiveRegion is passed', () => {
+  render(<HelperText isLiveRegion>text </HelperText>);
+  expect(screen.getByText('text')).toHaveAttribute('aria-live');
```

Comment:
```suggestion
  expect(screen.getByText('text')).toHaveAttribute('aria-live', 'polite');
```

Could we also do the inverse, checking aria-live doesn't render by default?